code/game/machinery/computer/dna_console.dm

```diff
@@ -562,6 +562,14 @@
 					viable_occupant.dna.unique_enzymes = buffer_slot["UE"]
 					viable_occupant.dna.blood_type = buffer_slot["blood_type"]
 
+	if(prob(1))
```

Comment:
This chance is extremely high for something that's supposed to stop spam. If you've ever played geneticist, you'd know how often you have to reapply buffers to yourself if you want powers. Corgization is pretty much irreversible without dying, so this is a pretty major nerf to normal genetics gameplay.

Reply:
Yeap